src/lib/loaders/loaders_without_authentication/vortex.ts

```diff
@@ -9,7 +9,10 @@ export default (opts) => {
   const { vortexLoaderWithoutAuthenticationFactory } = factories(opts)
 
   return {
-    vortexGraphqlLoader: ({ query, variables }: LoaderArgs) => {
+    vortexGraphqlLoaderWithoutAuthentication: ({
```

Comment:
You don't need to rename this, all the loaders are exposed in the context under the `loadersWithAuthentication` and `loadersWithoutAuthentication` objects already, so can pluck both/whichever ones you need. This is in addition to them being available at the root with the same-name collision.

That's in https://github.com/artsy/metaphysics/blob/aefe429bf964cd4ad90252dc98c06817b0ec4c7b/src/lib/loaders/index.ts#L93-L94